src/js/jsx/IconBar.jsx

```diff
@@ -110,6 +111,7 @@ define(function (require, exports, module) {
          */
         _handleBackToPSClick: function () {
             this.getFlux().actions.menu.native({ commandID: 5999 });
+            headlights.logEvent("UserInterface", "switch", "back-to-PS");
```

Comment:
For consistency, let's pick a particular syntax for all of these names and events. My suggestion: all lower case, all words separated by dashes. This will make our lives easier in the future.